app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -308,6 +323,15 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         }
     }
 
+    @VisibleForTesting(otherwise = PROTECTED)
+    internal open fun getIntentStartSource(intent: SafeIntent): Event.OpenedAppAllStart.Source? {
```

Comment:
nit: `internal` has no effect on visibility if you're not being consumed by another build module so we can remove this

Reply:
done